modules/formulize/include/formdisplay.php

```diff
@@ -241,85 +234,35 @@ public function renderValidationJS($withtags = true) {
 		return $js;
 	}
 
-    // reset will cause the cached copy of columns to be bypassed - this is done when a form is rendered, so that we don't reuse the setting for an element when it appeared on a different screen in the past in this same session
```

Comment:
Main culprit where we were storing and referencing stuff in session related to columns, now we can go off the screen object all the time